subsys/bluetooth/services/bas.c

```diff
@@ -0,0 +1,113 @@
+/** @file
+ *  @brief BLE GATT Battery Service
+ */
+
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <misc/byteorder.h>
+#include <misc/printk.h>
+#include <stdbool.h>
+#include <zephyr/types.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/services/bas.h>
+#include <bluetooth/uuid.h>
+
+#include <sensor.h>
+
+
+static struct bt_gatt_service bas_svc;
+static struct bt_gatt_attr attrs[];
+static struct bt_gatt_ccc_cfg blvl_ccc_cfg[BT_GATT_CCC_MAX];
+
+static struct device *battery;
+static struct sensor_value blvl = {0};
+static bool simulate_blvl;
```

Comment:
``notification_enabled`` would make more sense I think, since it is no simulation but potentially a real implementation.